cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/Aggregator.java

```diff
@@ -43,9 +43,12 @@
 
   /**
    * Aggregate all objects in the same group into zero or more output objects.
+   * If this aggregator does not implement {@link Reducer}, the group values will contain all input objects with
```

Comment:
Instead of overloading this method, I think it's better to have a `combineReducedValues` or `aggregateValues` method in the `Reducer` interface to make it clearer

Reply:
This is based on discussion with @albertshau . The `aggregator` API is basically same so we don't want to introduce a new interface as a new plugin. Maybe we can discuss further to finalize it